lib/webhooks/webhooks.d.ts

```diff
@@ -1,9 +1,9 @@
-import { Request, Response } from 'express';
-
-type Middleware = (
-  request: Request,
-  response: Response,
-  next: () => void
+type Request = (
```

Comment:
Think this should be an `interface` instead which the `express.Request` interface will implicitly implement.

Reply:
Something like this? 
```typescript
export interface Request {
  protocol: string;
  host: string;
  originalUrl: string;
  rawBody: Body;
  body: Body;
}
```

Reply:
Yes, but no need to export it since client will bring their own types (BYOT?).

`host` should be nested under `headers`, and needs a `header(string)` function.

Reply:
Can you clarify what you mean by the host needs a header(string) function? I see that host is nested in headers. Does this mean that I need to get the headers then search for the host?

Reply:
`header` function needed since we call `request.header('X-Twilio-Signature')`:
```
header(name: string): string | undefined;
```

We also call `request.headers.host` so need a `headers` property:
```
headers: IncomingHttpHeaders;
```